### PR TITLE
[DOC] clarify that the inertia_tensor may be nD in documentation

### PR DESCRIPTION
**Pull request by marshuang80**
## Description

Fixed import issue and documentation for inertia_tensor #4004

## Checklist

- [x] [Docstrings for all functions]
- [x] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: skimage/measure/_regionprops.py [hunk 1]
```diff
@@ -657,10 +657,10 @@ def regionprops(label_image, intensity_image=None, cache=True):
         bounding box.
     **image** : (H, J) ndarray
         Sliced binary region image which has the same size as bounding box.
-    **inertia_tensor** : (2, 2) ndarray
+    **inertia_tensor** : (Ndim, Ndim) ndarray
```
Comment (alexandrejaguar) [inline skimage/measure/_regionprops.py]: I think only mentioning this as `ndarray` should be sufficient:
```**inertia_tensor** : ndarray```

File: skimage/measure/_regionprops.py [hunk 2]
```diff
@@ -657,10 +657,10 @@ def regionprops(label_image, intensity_image=None, cache=True):
...
         Inertia tensor of the region for the rotation around its mass.
     **inertia_tensor_eigvals** : tuple
-        The two eigen values of the inertia tensor in decreasing order.
+        The N eigen values of the inertia tensor in decreasing order.
```
Comment (alexandrejaguar) [inline skimage/measure/_regionprops.py]: The `N` here is not necessary. It could be:
``` The eigenvalues of the inertia tensor in decreasing order.```

Comment (marshuang80) [reply skimage/measure/_regionprops.py]: Done!

**Comment by hmaarrfk** [comment]
Thanks!

**Pull request merged**